.github/workflows/publish-latest.yml

```diff
@@ -117,6 +117,10 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
+      - name: Trigger build for prod liveness sample app pointing at latest tag
+        run: curl -X POST -d {} $ENDPOINT -H "Content-Type:application/json"
+        env:
+          ENDPOINT: ${{ secrets.LIVENESS_PROD_SAMPLE_APP_BUILD_TRIGGER }}
```

Comment:
Theoretically this still might run even if liveness wasn't bumped and published. This is okay right?

Reply:
That's okay, it would just cause it to re-deploy witht he same version